package.json

```diff
@@ -1,10 +1,7 @@
 {
   "name": "cozy-ui",
-  "version": "0.1.7",
+  "version": "3.0.0",
   "description": "Cozy apps Ui SDK",
-  "scripts": {
```

Comment:
I don't get it. You removed the prepublish script but you didn't replaced it with a build and/or a watch. How is it supposed to build the CSS then? Or maybe i missed something else somewhere else. :open_mouth: